### PR TITLE
Fix styles for SearchPage

### PR DESCRIPTION
**Pull request by svitlua**

### CODE REVIEW

**Comment by svitlua** [comment]
[[Search Page] UI design doesn't meet mockup](https://trello.com/c/aBj7inv6/56-search-page-ui-design-doesnt-meet-mockup)

File: client/src/components/basic/checkbox/checkbox.module.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// TODO: replace font and colors to variables
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
+
+.formControlLabel {
+    font-size: 14px;
+    font-family: "Montserrat", sans-serif;
+    font-weight: 500;
+}
```
Comment (Paul-Vasylenko) [inline client/src/components/basic/checkbox/checkbox.module.scss:8]: EOF

File: client/src/components/pages/search-page/search-task/search-task.module.scss [hunk 2]
```diff
@@ -21,4 +9,12 @@
...
 	display: flex;
 	flex-direction: column;
+	margin-bottom: 20px;
+}
+
+h6 {
+	margin: 20px 0px 10px 0px;
+	font-family: "Montserrat", sans-serif;
+	font-weight: 600;
+	font-size: 16px;
```
Comment (Paul-Vasylenko) [inline client/src/components/pages/search-page/search-task/search-task.module.scss:19]: better use variables from the file.
If there is no correct variable for this - create a new one or use old and add some needed styles

File: client/src/components/pages/search-page/search-task/search-task.module.scss [hunk 3]
```diff
@@ -21,4 +9,12 @@
 .checkboxContainer {
 	display: flex;
 	flex-direction: column;
+	margin-bottom: 20px;
+}
+
+h6 {
+	margin: 20px 0px 10px 0px;
```
Comment (Paul-Vasylenko) [inline client/src/components/pages/search-page/search-task/search-task.module.scss:16]: 20px 0 10px

**Review by Paul-Vasylenko — CHANGES_REQUESTED** [review]

File: client/src/components/basic/checkbox/checkbox.module.scss (hunk 1 above)
Comment (Ev1ch) [reply client/src/components/basic/checkbox/checkbox.module.scss:5]: I am not sure that `font: $base-font-family;` will work, it is the shortcut for weight, size and family. Here you should use     `font-family: $base-font-family;`

**Review by Ev1ch — CHANGES_REQUESTED** [review]

File: client/src/components/basic/checkbox/checkbox.module.scss (hunk 1 above)
Comment (svitlua) [reply client/src/components/basic/checkbox/checkbox.module.scss:5]: thank you, fixed

**Review by Ev1ch — APPROVED** [review]
Looks good. Can be merged, I think

**Review by Ev1ch — APPROVED** [review]

**Review by Paul-Vasylenko — APPROVED** [review]

**Pull request merged**